soh/soh/Enhancements/randomizer/hook_handlers.cpp

```diff
@@ -1239,15 +1239,6 @@ void RandomizerOnVanillaBehaviorHandler(GIVanillaBehavior id, bool* should, void
             *should = INV_CONTENT((RAND_GET_OPTION(RSK_BOMBCHUS_IN_LOGIC) ? ITEM_BOMBCHU : ITEM_BOMB)) != ITEM_NONE;
             break;
         }
-        case VB_BLUE_WARP_APPLY_ENTRANCE_AND_CUTSCENE: {
```

Comment:
The issues with rando should be resolved as of https://github.com/HarbourMasters/Shipwright/pull/4340